### PR TITLE
first postmortem in blog

### PR DESCRIPTION
**Pull request by MacTynow**
This is the first postmortem to be publicly published. This is good content for the blog in my opinion.

### CODE REVIEW

File: _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  POST-MORTEM Production services downtime after a deployement
+author: Charles Martinot
+date:   2015-05-06 16:52:00
+---
+From now on, we will publish our post-mortems here. We hope that what we learn 
+from fixing downtimes and incidents will help other in turn.
+
+If you're not doing post-mortems yet, you should. Here is an [good article][1]
```
Comment (T-Dnzt) [inline _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown]: an -> a?

Comment (MacTynow) [inline _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown]: Right, at first I had written only 'an article'

File: _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+### Description of the issue : 
+Storage service and backends for all games were
+unreachable for between 5 and 30 minutes
+
+### Effect for the user : 
+All transactions with the backend were unable to be 
+processed (in-app purchases, loading and saving data...)
+
+### Solution found : 
+A environment variable was missing for one of our unused
```
Comment (T-Dnzt) [inline _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown]: An environment

**Comment by TamerShlash** [comment]
Cool, but I think there shouldn't be a space before the colon `:`

File: _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+author: Charles Martinot
+date:   2015-05-06 16:52:00
+---
+From now on, we will publish our post-mortems here. We hope that what we learn 
+from fixing downtimes and incidents will help others in turn.
+
+If you're not doing post-mortems yet, you should. Here is a [good article][1] 
+explaining why and how to do them.
+
+### Description of the issue :
```
Comment (TamerShlash) [inline _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown]: I think there shouldn't be a space before colons. (for this line an other lines).

Comment (T-Dnzt) [inline _posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown]: True, that's actually a French thing. It does look better with a space though so we all know what that means about French language.

**Pull request merged**